src/Features/CSharp/Portable/InheritanceMargin/CSharpInheritanceMarginService.cs

```diff
@@ -7,14 +7,13 @@
 using System.Collections.Immutable;
 using System.Composition;
 using System.Linq;
-using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Extensions;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.CodeAnalysis.InheritanceMargin;
 using Roslyn.Utilities;
 
-namespace Microsoft.CodeAnalysis.Editor.CSharp.InheritanceMargin
+namespace Microsoft.CodeAnalysis.CSharp.InheritanceMargin
 {
     [ExportLanguageService(typeof(IInheritanceMarginService), LanguageNames.CSharp), Shared]
     internal class CSharpInheritanceMarginService : AbstractInheritanceMarginService
```

Comment:
these types moved to Features level.  That way they can also be used by the remote service, while still utilizing inheritance (instead of statics).